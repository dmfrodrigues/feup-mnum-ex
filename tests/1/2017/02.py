def g(x, m, R):
    return x-x/m+R/(m*x**(m-1))
m = int(input())
R = float(input())
x = max(R,1.0)
i = 0
while i < 100:
    x_ = x
    x = g(x, m, R)
    print("%+.31f"%(x))
    if(x == x_): break
    i = i+1
