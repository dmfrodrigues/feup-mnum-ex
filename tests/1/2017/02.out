+50.5000000000000000000000000000000
+26.2400990099009909783944749506190
+15.0255301199868132044912272249348
+10.8404346730269249832190325832926
+10.0325785109606044898100662976503
+10.0000528956426926185940828872845
+10.0000000001398969828869667253457
+10.0000000000000000000000000000000
+10.0000000000000000000000000000000
